sp-installer-menu/menu/spi_sys_info.py

```diff
@@ -149,7 +149,7 @@ def main():
 
     if len(sys.argv) > 1 and sys.argv[1] == "nohtml":
         print()  # blank line
-        print(sep.join(out), file=sys.stdout)
+        print(linesep.join(out), file=sys.stdout)
```

Comment:
this was using "/" instead of newline when printing sysinfo to the terminal, oops